src/Summoner/Settings.hs

```diff
@@ -15,23 +15,23 @@ data CustomPrelude = CustomPrelude
 -- | Data needed for project creation.
 data Settings = Settings
     { settingsRepo           :: !Text   -- ^ repository name
-    , settingsOwner          :: !Text   -- ^ github username
+    , settingsOwner          :: !Text   -- ^ gitHub username
     , settingsDescription    :: !Text   -- ^ project description
     , settingsFullName       :: !Text   -- ^ full name
     , settingsEmail          :: !Text   -- ^ e-mail
     , settingsYear           :: !Text   -- ^ year
     , settingsCategories     :: !Text   -- ^ project category
     , settingsLicenseName    :: !LicenseName -- ^ type of license
     , settingsLicenseText    :: !License -- ^ license text
-    , settingsGithub         :: !Bool   -- ^ github repository
+    , settingsGitHub         :: !Bool   -- ^ gitHub repository
     , settingsTravis         :: !Bool   -- ^ Travis CI integration
-    , settingsAppVey         :: !Bool   -- ^ AppVeyor CI integration
+    , settingsAppVeyor       :: !Bool   -- ^ AppVeyor CI integration
```

Comment:
@cdeepanshu Thanks for starting to work on this! I see that CI if failing. Looks like you need to update all places where these variables are used, not only the data type definition.

Reply:
ok i will do that